src/components/features/NewFurniture/NewFurniture.js

```diff
@@ -12,6 +13,7 @@ class NewFurniture extends React.Component {
     activePage: 0,
     activeCategory: 'bed',
     className: styles.fadeEnd,
+    activeCategorySales: 'top seller',
```

Comment:
takie rzeczy jak nazwa kategori `top seller` powinny być zapisane w jakiś stałych, aby nie zrobić literówki

Reply:
Ja zmieniłem kompletnie ten komponent na klasowy, teraz jest to przygotowane pod przyszłe dodanie działania :) top seller teraz jest już jako stała, przejrzyj proszę najnowsze zmioany, bo jak dodawałeś komentarze, to jeszcze było przed zmianą tego na klasowy pod przyszła funkcjonalność :) Sorka za zamieszanie Kamilu ! :)